### DISCUSSION TITLE
Possible issue with IPv6

### DISCUSSION THREAD

**Discussion by robertlynch3** [chuot/rdio-scanner#224] [General]
I am opening this as a discussion because I don't know if its an issue with my reverse proxy or Rdio-scanner. 

When I connect to Rdio-Scanner via IPv6, log messages show `new listener from ip 2601` rather than the entire IP address. Is anyone else seeing this issue or is it just my setup?

**Comment by chuot** [comment]
It's clearly a bug in the regular expression I'm using. 2601 is basically the first part of your IPV6 address. Please open an issue, I'll make sure to fix it before v6.5. Thanks!

**Reply by robertlynch3** [reply]
Issue created. I wasn't sure if it was my setup because I have a complex reverse proxy via Cloudflare, then an external machine, then IPv4 to the host that runs the docker container. 

Let me know if you find anything, I feel like it still could be my own setup!
